packages/react/src/css/index.tsx

```diff
@@ -8,12 +8,19 @@ import type { CSSProps } from '../types';
  * css`color: red;`;
  * ```
  *
+ * ```
+ * css({ color: 'red' });
```

Comment:
Although they're missing right now, you should add tests for this usage under `packages/react/src/css/__tests__/index.test.tsx` that utilise `toHaveCompiledCss`